doc/Makefile

```
@@ -36,6 +37,15 @@ html: api
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+html-noplot: api
+	@echo
+	@echo "Building HTML files."
```

```suggestion
	@echo "Building HTML files without example plots."
```
I think it's good to have a different print statement than `make html`.